rebar.config

```diff
@@ -4,6 +4,7 @@
     {pbkdf2, {git, "git://github.com/marianoguerra/erlang-pbkdf2-no-history", {branch, "master"}}},
     {exometer_core, {git, "git://github.com/basho/exometer_core.git", {branch, "th/correct-dependencies"}}},
     {riak_core, {git, "git://github.com/lasp-lang/riak_core", {branch, "partisan-support-r21"}}},
+    {cuttlefish, {git,"git://github.com/kyorai/cuttlefish.git", {branch, "develop"}}},
```

Comment:
this should probably be a tag or master or something stable-ish, will change that 🤦‍♂